instructor/templates/instructor/course_setup.html

```diff
@@ -0,0 +1,95 @@
+{% extends "instructor/assignment_sidebar.html" %}
+
+{% block right_side %}
+    <style>
+        .scb_f_use_existing_course {
+            display: none;
+        }
+    </style>
+    <script>
+        $(function () {
+            $(".scb_f_course_setup_save_continue_course_button").click(function () {
+                var course_pk = $("option:selected").val();
+                var data = {course_pk: course_pk};
+                if ($(this).attr('name') === 'save') {
+                    data['save'] = 'save';
+                } else {
+                    data['continue'] = 'continue'
+                }
+                var url = '/ab/assignments/course_setup/';
+                if ("{{new}}" === 'False') {
+                    url = '/ab/assignments/course_modify/'
+                }
+                $.ajax({
+                    url: url,
+                    type: 'POST',
+                    data: data
```

Comment:
you need some error handling of some sort on this post unless I'm missing something